src/pages/LoginPage.tsx

```
@@ -1,15 +1,57 @@
+  }
+
+  const handleLogin = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+  
+    const model: loginModel = {
+      email,
+      password
+    };
+  
+    try {
+      const response:any = await login({data:{payload:model}}).unwrap(); 
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)